lib/migration.js

```diff
@@ -14,6 +14,17 @@ const { confirm, STATE_SUCCESS, STATE_FAILURE } = require('./config');
 
 const { storeMigration, getNewMigrations } = require('./backend');
 
+const migrationHeader = stripIndent`/* eslint-env node */
+  const { withHelpers } = require('@jungvonmatt/contentful-migrations');
+
+  /**
+   * Contentful migration
+   * API: https://github.com/contentful/contentful-migration
+   * Editor Interfaces: https://www.contentful.com/developers/docs/extensibility/app-framework/editor-interfaces/
```

Comment:
Wo wir hier diese Links haben, sollten wir da auch einen Link zu einer Readme bezüglich der util functions linken?
Zu den Utils sollte auch noch was in der haupt readme gesagt werden

Reply:
I added a section to the readme concerning the helper functions. I wouldn't add a list here because we have type hints available for the helpers. The links provide just a bit more information about how to write migrations.